stage1/init/common/pod.go

```diff
@@ -256,28 +259,61 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		return errwrap.Wrap(errors.New("unable to write environment file"), err)
 	}
 
-	// This is a partial implementation for app.User and app.Group:
-	// For now, only numeric ids (and the string "root") are supported.
-	var uid, gid int
+	var _uid, gid int
```

Comment:
Otherwise it conflicts with the `uid` package since `uid.NewBlankUidRange()` is called shortly after. Are there any preferred naming rules in situations like these?